web/src/pages/guidance/published.module.scss

```diff
@@ -0,0 +1,14 @@
+@use '../../../node_modules/@nice-digital/nds-core/scss/core';
+
+.shortTime {
+  @include core.mq($from: md) {
```

Comment:
Oooh that's interesting the core module is being exported as a 'proper' sass module, very nice